corehq/motech/dhis2/static/dhis2/js/dataset_map.js

```diff
@@ -11,6 +11,8 @@ hqDefine("dhis2/js/dataset_map", [
 ) {
     $(function () {
         var $sendNowResult = $('#send-now-result');
+        var $remoteLogsLink = $('#remote-logs');
```

Comment:
nit: I think we normally try and bundle the `var` declarations
```suggestion
        var $sendNowResult = $('#send-now-result'),
            $remoteLogsLink = $('#remote-logs');
```